### PR TITLE
Migrate em namespace to v2

### PR DESCRIPTION
**Pull request by adusumillipraveen**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: apps/em/aat/00/kustomization.yaml [hunk 1]
```diff
@@ -3,3 +3,5 @@ kind: Kustomization
 resources:
   - ../base
 namespace: em
+patchesStrategicMerge:
+  - ../../em-hrs-ingestor/schedule-cluster-0.yaml
```
Comment (thomast1906) [inline apps/em/aat/00/kustomization.yaml:7]: can you update naming?

File: apps/em/aat/01/kustomization.yaml [hunk 2]
```diff
@@ -3,3 +3,5 @@ kind: Kustomization
 resources:
   - ../base
 namespace: em
+patchesStrategicMerge:
+  - ../../em-hrs-ingestor/schedule-cluster-1.yaml
```
Comment (thomast1906) [inline apps/em/aat/01/kustomization.yaml:7]: can you update naming?

Comment (adusumillipraveen) [reply apps/em/aat/01/kustomization.yaml:7]: They are using across envs to avoid duplication. Will leave it for now.

**Review by thomast1906 — DISMISSED** [review]

File: apps/em/perftest/base/kustomization.yaml [hunk 3]
```diff
@@ -8,4 +8,7 @@ patchesStrategicMerge:
   - ../../identity/perftest.yaml
   - ../../em-ccd-orchestrator/perftest.yaml
   - ../../em-anno/perftest.yaml
-  - ../../em-hrs-ingestor/schedule-off.yaml
\ No newline at end of file
+  - ../../em-hrs-ingestor/schedule-off.yaml
+  - ../..//em-stitching/perftest.yaml
+  - ../..//em-npa/perftest.yaml
```
Comment (thomast1906) [inline apps/em/perftest/base/kustomization.yaml:13]: typo `//`

File: apps/em/prod/base/kustomization.yaml [hunk 4]
```diff
@@ -7,3 +7,7 @@ patchesStrategicMerge:
   - ../../identity/prod.yaml
   - ../../em-ccd-orchestrator/prod.yaml
   - ../../em-anno/prod.yaml
+  - ../../em-hrs-ingestor/prod.yaml
+  - ../..//em-npa/prod.yaml
```
Comment (thomast1906) [inline apps/em/prod/base/kustomization.yaml:11]: typo `//`

**Pull request merged**